guides/common/modules/con_configuring-project-to-use-an-alternative-directory-to-execute-remote-jobs-on-hosts.adoc

```diff
@@ -0,0 +1,15 @@
+[id="configuring-project-to-use-an-alternative-directory-to-execute-remote-jobs-on-hosts_{context}"]
+= Configuring {Project} to use an alternative directory to execute remote jobs on hosts
+
+ifeval::["{context}" == "managing-hosts"]
+By default, {Project} uses the `/run` directory on the client for the remote execution jobs.
+endif::[]
+ifeval::["{context}" == "ansible"]
+Ansible puts its own files it requires into the `$HOME/.ansible/tmp` directory, where `$HOME` is the home directory of the remote user.
+endif::[]
+You can configure {Project} to use an alternative directory if required.
+
```

Comment:
I recommend adding a use case. What circumstances would require configuring an alternative directory? Are there any advantages or disadvantages?

Reply:
Currently trying to find out myself with the help of @adamruzicka.

Reply:
But yes we definitely need a usecase.